src/site/content/en/blog/badging-api/index.md

```diff
@@ -71,7 +70,7 @@ Examples of sites that may use this API include:
 | 1. Create explainer                        | [Complete][explainer]        |
 | 2. Create initial draft of specification   | [Complete][spec]             |
 | 3. Gather feedback & iterate on design     | [In progress](#feedback)     |
-| **4. Origin trial**                        | [**In progress**](#ot)       |
+| **4. Origin trial**                        | v2 (Chrome 79-80) [In progress](#ot)<br>v1 (Chrome 73-78) [In progress](#backwards-compat) |
```

Comment:
@mgiuca is this trial definitely finishing in M80?

Reply:
Currently it's scheduled for the standard 3 milestones (79-81). We might cut it short but for now, just say 81 here.

Reply:
Updated.